cg/services/order_validation_service/models/errors.py

```diff
@@ -155,6 +155,17 @@ class SubjectIdSameAsSampleNameError(CaseSampleError):
     message: str = "Subject id must be different from the sample name"
 
 
+class InvalidConcentrationIfSkipRCError(CaseSampleError):
+    def __init__(self, case_name: str, sample_name: str, allowed_interval: tuple[int, int]):
+        field: str = "concentration_ng_ul"
+        message: str = (
+            f"Concentration must be between {allowed_interval[0]} ng/μL and {allowed_interval[1]} ng/μL"
```

Comment:
Probably relevant to add this 
```suggestion
            f"Concentration must be between {allowed_interval[0]} ng/μL and {allowed_interval[1]} ng/μL if reception control should be skipped"
```

Reply:
Nice catch!